screenpy/speech_tools.py

```diff
@@ -52,22 +52,24 @@ class name: replace each capital letter with a space and a lower-case
 
 
 @overload
-def represent_prop(item: str) -> str: ...
+def represent_prop(item: mock.Mock) -> mock.Mock: ...
 
 
 @overload
-def represent_prop(item: T) -> T: ...
+def represent_prop(item: str | T) -> str: ...
 
 
-def represent_prop(item: str | T) -> str | T:
+def represent_prop(item: str | T | mock.Mock) -> str | mock.Mock:
     """Represent items in a manner suitable for the audience (logging)."""
-    if not ismock(item) and hasmethod(item, "describe_to"):
+    if isinstance(item, mock.Mock):
+        return item
+    if hasmethod(item, "describe_to"):
         return f"{item}"
     if isinstance(item, str):
         return repr(item)
 
     description = str(item)
     if description[:1] == "<" and description[-1:] == ">":
```

Comment:
I don't even remember what this bit was in here for, but cab we update this check to do `description[0]` and `description[-1]`? 

Haha, what's the deal with the slicing? 🤔

Reply:
The basic idea behind `represent_prop` is to wrap most objects in `< >`, but avoid adding a second set on `str` output that already contains them.

Example:
mock objects would look like `<<Mock>>`; which looks silly.  

Since it was impossible to know which objects might have `< >` in their str output, we have to do a slice check.
Although, as I type this I suppose I could use `.startswith` and `.endswith`.  (committing that change)

Reply:
Ahh, I forgot that slicing an empty string with `[:1]` avoids `IndexError` where `[0]` does not.
Still, `startswith` is a better solution.